BlockSettleSigner/SignerAdapterListener.cpp

```diff
@@ -851,6 +851,7 @@ bool SignerAdapterListener::onImportWoWallet(const std::string &data, bs::signer
    const auto woWallet = walletsMgr_->loadWoWallet(settings_->netType()
       , settings_->getWalletsDir(), request.filename(), app_->controlPassword());
    if (!woWallet) {
+      SystemFileUtils::rmFile(filePath);
```

Comment:
Why do we need to remove any data? BST-2729 doesn't require this.

Reply:
We create new wallet file in wallets dir and try to load it. So it's better to remove file if import fails for any reason.